hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/AssertionSinks.java

```diff
@@ -138,6 +146,46 @@ private AssertionSinks() {
             .build();
     }
 
+    /**
+     * Asserts that the previous stage emitted the exact sequence of
+     * expected items and nothing else.
+     * <p>
+     * Is just a wrapper around {@link #assertCollectedEventually(int, ConsumerEx)},
```

Comment:
javadoc needs to be clear and doens't need to list the implementation details. see the other methods' javadoc.